modules/xmpp/ChatRoom.js

```diff
@@ -514,6 +532,14 @@ export default class ChatRoom extends Listenable {
                 this.recording.handleJibriPresence(this.lastJibri);
             }
         }
+
+        if (!this.videoSIPGWHandler) {
```

Comment:
Why is VideoSIPGW inside of the ChatRoom ? It feels like the ChatRoom should be dealing with XMPP MUC stuff and not managing other components.